### ISSUE TITLE
Ability to query whats going on

### ISSUE THREAD

**Issue by whyrusleeping** [filecoin-project/go-sectorbuilder#3]
It would be nice to have some exposed apis that allow the user to query what the sectorbuilder is currently doing, like sealing N sectors in parallel, 65% complete, etc. Some sort of thing we could use to inspect why my cpu is melting.

I'm not exactly sure what i want from these apis, but i know we need something

**Comment by magik6k** [comment]
I'll try to make a storage miner view in pond after the weekend
